test/feature/detect_victor_test.exs

```diff
@@ -11,14 +11,20 @@ defmodule DetectVictorTest do
   end
 
   test "declares X is the winner" do
-    assert Board.winner("XXXOO6789", "X", "Y", "O", "U") == "Y"
+    player_one = %Player{name: "X", marker: "X"}
```

Comment:
**thought:** is there a way you can hide having to know the struct specifics? If you change your `Player` struct that's a lot of tests to change.